plugins/newsdownloader.koplugin/main.lua

```diff
@@ -160,12 +160,13 @@ function NewsDownloader:loadConfigAndProcessFeeds()
     for idx, feed in ipairs(feed_config) do
         local url = feed[1]
         local limit = feed.limit
+        local download_full_article = feed.download_full_article == "true" or feed.download_full_article == true
```

Comment:
I'm not sure if there's a need for `"true"`, but if there is you should just do this:

```lua
local download_full_article = tostring(feed.download_full_article) == "true" or false
```
I'd probably just go for this.
```lua
local download_full_article = feed.download_full_article or false -- or true I suppose
```

Reply:
type(feed.download_full_article) return boolean so it's save to leave it:
local download_full_article = feed.download_full_article